set-env.sh

```diff
@@ -14,4 +14,5 @@ export COMPOSE_HTTP_TIMEOUT=240
 export EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS=$DOCKER_HOST_IP:9092
 export EVENTUATELOCAL_CDC_DB_USER_NAME=root
```

Comment:
Are any of these env vars necessary?
In the SPring application the application.properties use `${DOCKER_HOST_IP:localhost}`

Reply:
EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS - probably used by tests.
COMPOSE_HTTP_TIMEOUT - sometimes ago solved some issue, I don't remembed what it was